Library/Formula/wxmac.rb

```diff
@@ -15,6 +15,8 @@ class Wxmac < Formula
   depends_on "jpeg"
   depends_on "libpng"
   depends_on "libtiff"
+  
+  option "with-enable-stl", "use standard C++ classes for everything"
```

Comment:
Just call this `with-stl`. Any reason to not enable it by default, out of interest?